src/app/works/[id]/page.tsx

```
@@ -23,7 +23,20 @@ const WorkDetailPage = () => {
-  return <></>;
+  return (
+    <>
+      <WorkTemp
+        id={work.id}
+        client={work.client}
+        title={work.title}
+        desc={work.short_desc}
+        link={work.site_link}
+        img={work.top_image.url}
+      >
+        <div dangerouslySetInnerHTML={{ __html: work.content }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)